src/thirdparty/caman/caman.full.min.js

```diff
@@ -0,0 +1,244 @@
+/*
```

Comment:
How come you didn't just use `npm` to install this vs. pulling it directly into the tree?  Not a blocker, but seems like something you could have done--other parts of Brackets do this now.